src/dlgConnectionProfiles.cpp

```diff
@@ -1697,6 +1709,11 @@ bool dlgConnectionProfiles::validateProfile()
 {
     bool valid = true;
 
+    // don't validate url duplication during copy, as information will already exist when we try to set it
+    if (mCopyingProfile) {
```

Comment:
Is there a potential problem with atomicity with this flag? if there is a background copy going on and the user selects a different profile, won't that trigger a run of this method with the need for this flag to NOT be set...

Do we need to not allow the profile selection to be changed while the copy is taking place? Ah, new line 1443 will already have reset the flag before it runs this method directly - but I'm guessing that it also causes for it to indirectly to be run earlier as well...

Reply:
It is not a concern, the `copy` button will stay disabled until the copy is finished:


https://user-images.githubusercontent.com/110988/162424829-301e22f9-ec39-4052-a051-f88336af4e4f.mp4

Reply:
:+1: Okay!